lib/event_sources.json.example

```diff
@@ -11,7 +11,8 @@
     {
       "ScheduleName": "node-lambda-test-schedule",
       "ScheduleState": "ENABLED",
-      "ScheduleExpression": "rate(1 hour)"
+      "ScheduleExpression": "rate(1 hour)",
+      "ScheduleDescription": "Run node-lambda-test-function once per hour"
```

Comment:
@EnriqueCanals that would be great! Thanks for taking the time to make this project better!